### DISCUSSION TITLE
MacOS 9: How to install .sea file?

### DISCUSSION THREAD

**Discussion by rdreasn** [utmapp/UTM#4359] [UTM Help (macOS)]
I have downloaded and install MacOS 9.  Now I have downloaded [this app](https://www.macintoshrepository.org/1158-dabbler-2-0), it's .sea file.  How do I install it to the MacOS 9 in UTM?

**Comment by easp** [comment]
Have you tried double-clicking?

**Comment by rdreasn** [comment]
I can't even put the .sea file into the MacOS 9, so I can't access it from MacOS 9.  
How do I transfer file from host OS (Monterey) to the VM?
